integration/helpers/utils.ts

```diff
@@ -110,6 +110,7 @@ export async function tearDown(app: Application) {
 
   try {
     process.kill(pid, "SIGKILL");
+    spawnSync("killall lens");
```

Comment:
Will this kill my Lens 4.0.8 that I might have open while running the integration tests?

Reply:
Yes, thanks for reminding me. I added this for testing purposes and should remove it.